### ISSUE TITLE
Yahka über VPN verbundende Netzwerke?

### ISSUE THREAD

**Issue by hohmannc85** [jensweigele/ioBroker.yahka#370]
Moin! Vllt etwas ungewöhnliche Frage, aber vllt könnt ihr mir helfen.
Meine Eltern haben kein Smarthome zentrale.
Ich habe unsere FRITZ!Boxen mit vpn verbunden
Eltern 192.168.178.x ich 192.168.188.x
Ich empfange über den iobroker schon mittels mqtt die Daten meiner Eltern. Wenn ich eine zweite yahka Bridge erstelle, taucht diese jedoch nur bei mir auf wenn ich sie im Netzwerk in HomeKit suche. Gibt es eine Möglichkeit, dass diese auch im Netz meiner Eltern gefunden werden?
Wahlweise müssten sonst meine Eltern zu mir um einmalig die Bridge zu finden?
Ich bin auf die Antwort gespannt! Danke für den super Adapter!

**Comment by c1328** [comment]
Hi! Gibts hier inzwischen eine Lösung? Mich würde interessieren, wie sich der HomeKit Adapter finden lässt, wenn der ioBroker via VPN in das Heimnetz eingebunden ist.

**Comment by hohmannc85** [comment]
Stichwort mDNS Tunnel

**Comment by c1328** [comment]
Danke! Zum Auffinden hat es funktioniert, mehr aber auch nicht. Scheint aber ios hat da grad eh ein Thema mit HomeKit....

**Comment by hohmannc85** [comment]
Hab’s auch nicht umgesetzt weil man ja doch 2 Geräte auf beiden Seiten brauch und dann brauch ich es nicht ;-)

**Comment by c1328** [comment]
Ich habe der Sache nochmal eine Chance gegeben und es scheint, dass der mDNS Tunnel nur zur Bekanntmachung der yahka devices benötigt wird - vorausgesetzt man hat noch ein iPad als Steuerzentrale konfiguriert (ob das zwingend notwendig ist kann ich nicht sagen, aber ich wollte eh die iCloud Funktionalität). Inzwischen ist der Tunnel längst wieder deaktiviert und es funktioniert über die HomeKit App einwandfrei (Heizungsthermostate und ein paar Shellies).
Architektur: iobroker als VM auf Proxmox (im Rechenzentrum) hängt über VPN an einer Fritz!Box (kein weiterer Server/Raspi o.ä. im Heimnetz). Ob das mein finales Setup ist? Würde dafür meine Hand nicht ins Feuer legen ;-)

**Comment by c1328** [comment]
....zu früh gefreut, Verbindung geht verloren. Muss also eine andere Lösung her.....

**Comment by hohmannc85** [comment]
Also das was du vor hast, steuern über HomeKit benötigst du ja keinen Tunnel oder vpn, sondern eine steuerzentrale. Alte iPads gehen nicht mehr. Also homepod oder Apple TV nehmen. Die vpn Geschichte wollte ich benutzen um ein anderes zu Hause über vpn auch Smart zu machen ohne dass die ein pi benötigen

**Comment by c1328** [comment]
Aber das ist doch genau mein Szenario. Eine ioBroker VM hängt über VPN im Heimnetzwerk und soll dieses Smart machen :-) Der Unterschied zu dir ist eigentlich nur: es gibt kein zweites Heimnetzwerk. Ein IPad ist zusätzlich als Steuerzentrale hinterlegt - aber das ist ein anderes Thema und für das eigentliche Ziel nicht relevant.

**Comment by hohmannc85** [comment]
Ja ok stimmt, aber dann brauchst du in deinem Heimnetzwerk ja den Tunnel auf dein vpn.
Aber wenn die vm eh in deinem Netzwerk ist, warum verbindest du die per VPN und nicht direkt, dann gebe es das Problem doch nicht? Oder ist die vm physisch an einem anderen ort

**Comment by c1328** [comment]
Die VM läuft im Rechenzentrum.

**Comment by hohmannc85** [comment]
Und dann hat du einen pi im Heimnetzwerk für den mdns Tunnel richtig?

**Comment by c1328** [comment]
Genau, temporär für den Tunnel habe ich ein altes Linux Notebook ertüchtigt - aber Ziel war/ist, eben keinen weiteren Server/Raspi etc. im Heimnetz zu haben.
